build.gradle.kts

```diff
@@ -273,17 +273,6 @@ nexusPublishing {
     }
 }
 
-githubRelease {
```

Comment:
please remove the plugin from build.gradle and settings.gradle and remove the version from gradle.properties